src/main/java/org/sagebionetworks/WorkflowAdmin.java

```diff
@@ -81,6 +82,11 @@ public static void main( String[] args ) throws Throwable {
 	
 	public SynapseClient getSynapseClient() {return synapseAdmin;}
 
+	public WorkflowAdmin(SynapseClient synapseAdmin, Archiver archiver) throws SynapseException {
```

Comment:
We can remove the throw. We could also remove the empty constructor and login in the main.